build_release/Handle_OBS_results.py

```diff
@@ -339,7 +339,7 @@ def Get_packages_on_OBS():
                    fnmatch.fnmatch(Distrib_name, "Fedora*") or \
                    fnmatch.fnmatch(Distrib_name, "Arch*"):
                     Bin_or_lib_name = Devel_name
-                elif Distrib_name == "xUbuntu_15.10" or Distrib_name == "xUbuntu_16.04":
+                elif Distrib_name == "xUbuntu_15.10" or Distrib_name == "xUbuntu_16.04" or Distrib_name == "Ubuntu_Next_Standard":
```

Comment:
We'll forget for every new version that we need to add "v5"
Not possible e.g. >15.10 or Next?